frontend_app_file/src/component/FileComponent.jsx

```diff
@@ -35,9 +35,7 @@ export class FileComponent extends React.Component {
     const videoWrapper = document.createElement('div')
     videoWrapper.id = 'videoWrapperDiv'
     videoWrapper.className = 'file__previewVideo open'
-    videoWrapper.onclick = (e) => {
-      this.props.onClickClosePreviewVideo(e)
-    }
+    videoWrapper.onclick = () => this.props.onTogglePreviewVideo()
```

Comment:
this is equivalent to:
`videoWrapper.onclick = this.props.onTogglePreviewVideo`
unless you specifically don't want to pass the event object

Reply:
Fixed